explainaboard/analysis/case.py

```diff
@@ -80,6 +81,36 @@ class AnalysisCaseLabeledSpan(AnalysisCaseSpan):
     predicted_label: str
 
 
+@dataclass
+class AnalysisCaseBlock(AnalysisCase):
+
+    text: str
+    orig_str: str
+
+
+@dataclass
+class AnalysisCaseLabeledBlock(AnalysisCaseBlock):
```

Comment:
Suggestion: could we rename this `AnalysisCaseArgumentPair`? The fields in here such as `block_review_sentences` and `block_reply_sentences` are specific enough that I don't think this can be applied to anything other than the APE task, so the naming should probably reflect this as well. It'd be nice to update the docstring to also mention the APE task.